src/applications/simple-forms/40-0247/tests/e2e/0247-pmc.cypress.spec.js

```diff
@@ -34,100 +75,49 @@ const testConfig = createTestConfig(
+                  'applicantAddress',
+                  data.applicantAddress,
+                );
+
+                cy.axeCheck('.form-panel');
+                cy.findByText(/continue/i, { selector: 'button' }).click();
+              });
           });
         });
       },
-      [pagePaths.veteranIdentificationInfoPage]: ({ afterHook }) => {
-        afterHook(() => {
-          cy.get('@testData').then(data => {
-            fillTextWebComponent('veteranId_ssn', data.veteranId.ssn);
-
-            cy.axeCheck('.form-panel');
-            cy.findByText(/continue/i, { selector: 'button' }).click();
-          });
-        });
-      },
-      [pagePaths.requestTypePage]: ({ afterHook }) => {
-        afterHook(() => {
-          cy.get('@testData').then(data => {
-            selectYesNoWebComponent('isFirstRequest', data.isFirstRequest);
-
-            cy.axeCheck('.form-panel');
-            cy.findByText(/continue/i, { selector: 'button' }).click();
-          });
-        });
-      },
-      [pagePaths.applicantPersonalInfoPage]: ({ afterHook }) => {
-        afterHook(() => {
-          cy.get('@testData').then(data => {
-            fillFullNameWebComponentPattern(
-              'applicantFullName',
-              data.applicantFullName,
-            );
-
-            cy.axeCheck('.form-panel');
-            cy.findByText(/continue/i, { selector: 'button' }).click();
-          });
-        });
-      },
-      [pagePaths.applicantAddressPage]: ({ afterHook }) => {
-        afterHook(() => {
-          cy.get('@testData').then(data => {
-            fillAddressWebComponentPattern(
-              'applicantAddress',
-              data.applicantAddress,
-            );
-
-            cy.axeCheck('.form-panel');
-            cy.findByText(/continue/i, { selector: 'button' }).click();
-          });
-        });
-      },
-      [pagePaths.applicantContactInfoPage]: ({ afterHook }) => {
-        cy.injectAxeThenAxeCheck();
-        afterHook(() => {
-          cy.get('@testData').then(data => {
-            fillTextWebComponent('applicantPhone', data.applicantPhone);
-
-            cy.axeCheck('.form-panel');
-            cy.findByText(/continue/i, { selector: 'button' }).click();
-          });
-        });
-      },
-      [pagePaths.additionalCertificatesRequestPage]: ({ afterHook }) => {
+      'additional-certificates-request': ({ afterHook }) => {
         afterHook(() => {
           cy.get('@testData').then(data => {
             const { additionalAddress, additionalCopies } = data;
 
-            fillAddressWebComponentPattern(
-              'additionalAddress',
-              additionalAddress,
-            );
-            fillTextWebComponent('additionalCopies', additionalCopies);
-
-            cy.axeCheck('.form-panel');
-            cy.findByText(/continue/i, { selector: 'button' }).click();
+            cy.get('input[name="root_additionalAddress_state"]')
+              .should('not.be.disabled')
```

Comment:
and here

Reply:
Yeah, I think the page-specific test-block here is the right place to assert on a specific field.  I've also removed the tagname (or element-type) from my selector above, so the assertion works on both text-inputs & selects.